src/EDI/Parser.php

```diff
@@ -40,16 +39,34 @@ class Parser
      * @var string : end character (default ')
      */
     private $symb_end;
+    /**
+     * @var string : encoding (default UNOA)
+     */
+    private $encoding;
     
+    private $encodintToStripChars = [
+        "UNOA" => "/[\x01-\x1F\x80-\xFF]/", // not as restrictive as it should be
```

Comment:
If we set default UNOB, I think that perhaps the UNOA should be the restrictive version.